packages/material-ui/src/Modal/ModalManager.js

```diff
@@ -148,7 +148,7 @@ class ModalManager {
     // If that was the last modal in a container, clean up the container.
     if (data.modals.length === 0) {
       if (this.handleContainerOverflow) {
-        removeContainerStyle(data);
+        this.removeAdjustedStyles = () => removeContainerStyle(data);
```

Comment:
Doing it here means we can access data easier without having references in `ModalManager`